Add "Making an offer" section to People Ops docs (#1157)
sourcegraph/about: handbook/people-ops/hiring.md

1. What is the current runway and the monthly burn rate?
1. How much cash does the company have in the bank?
1. Other [questions about Sourcegraph's internal financial metrics](../ceo/index.md#questions-about-sourcegraph).

## Making an offer

Once the decision is made to [extend an offer](interview_process.md#outcome) to a candidate, the hiring manager and a representative from the People Ops team will schedule a call to walk through a summary of the offer. This summary will include an overview of cash compensation (including any commission or bonus plans), benefits, and any equity-based compensation (including a simple calculator for potential future values).

> With the candidate's approval, Sourcegraph will then conduct three reference check interviews with former peers and at least one former manager. If these all come back positive, the Sourcegraph legal team will send the official DocuSign with the appropriate contracting or employment agreement(s) for the candidate's role, status, and location. Templates for employees and contractors can be found in the shared [HR/Employees/Contractors Dropbox folder](https://www.dropbox.com/home/Legal/HR%3AEmployees%3AContractors).

Reviewer: I am worried that `If these all come back positive` implies that one bad reference will necessarily tank your offer.

```suggestion
With the candidate's approval, Sourcegraph will then conduct three reference check interviews with former peers and at least one former manager. If we are happy with the results, the Sourcegraph legal team will send the official DocuSign with the appropriate contracting or employment agreement(s) for the candidate's role, status, and location. Templates for employees and contractors can be found in the shared [HR/Employees/Contractors Dropbox folder](https://www.dropbox.com/home/Legal/HR%3AEmployees%3AContractors).
```

Author: Thanks, great call
